frontend/src/components/Dashboard.js

```diff
@@ -0,0 +1,223 @@
+    }
+  });
+
+  return (
+    <div>
+      <h1 class="text-4xl font-bold mb-4 text-red-500">
+        Welcome {store?.data?.full_name || store?.data?.username}!
+      </h1>
+      <div
+        id="accordion-flush"
+        data-accordion="collapse"
+        data-active-classes="bg-white dark:bg-gray-900 text-gray-900 dark:text-white"
+        data-inactive-classes="text-gray-500 dark:text-gray-400"
+      >
+        <Show when={store.data.player}>
+          <h2 id="accordion-heading-player">
+            <button
+              type="button"
+              class="flex items-center justify-between w-full py-5 font-medium text-left text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400"
+              data-accordion-target="#accordion-body-player"
+              aria-expanded="false"
+              aria-controls="accordion-body-player"
+            >
+              <span>Player Information: {store.data.player.full_name}</span>
+              <AccordionDownIcon />
+            </button>
+          </h2>
+          <div
+            id="accordion-body-player"
+            class="hidden"
+            aria-labelledby="accordion-heading-player"
+          >
+            <div class="py-5 border-b border-gray-200 dark:border-gray-700">
+              <Player player={store.data.player} />
+            </div>
+          </div>
+        </Show>
+        <Show when={store.data.wards}>
+          <For each={store.data.wards}>
+            {ward => (
+              <>
+                <h2 id={`accordion-heading-ward-${ward.id}`}>
+                  <button
+                    type="button"
+                    class="flex items-center justify-between w-full py-5 font-medium text-left text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400"
+                    data-accordion-target={`#accordion-body-ward-${ward.id}`}
+                    aria-expanded="false"
+                    aria-controls={`accordion-body-ward-${ward.id}`}
+                  >
+                    <span>Player Information: {ward.full_name}</span>
+                    <AccordionDownIcon />
+                  </button>
+                </h2>
+                <div
+                  id={`accordion-body-ward-${ward.id}`}
+                  class="hidden"
+                  aria-labelledby={`accordion-heading-ward-${ward.id}`}
+                >
+                  <div class="py-5 border-b border-gray-200 dark:border-gray-700">
+                    <Player player={ward} />
+                  </div>
+                </div>
+              </>
+            )}
+          </For>
+        </Show>
+        <h2 id="accordion-heading-actions">
+          <button
+            type="button"
+            class="flex items-center justify-between w-full py-5 font-medium text-left text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400"
+            data-accordion-target="#accordion-body-actions"
+            aria-expanded="false"
```

Comment:
Expanded user actions accordion if user hasn't done any registration stuff was implemented in abff00026592d3c4503a1936246fc06e681c694d. Again, not sure why this was changed. :)

Reply:
Same as [here](https://github.com/india-ultimate/hub/pull/75#discussion_r1309637177)